Library/Homebrew/cmd/bottle.rb

```diff
@@ -281,7 +281,8 @@ def bottle_formula(f)
       bottle.prefix prefix
     end
     bottle.revision bottle_revision
-    bottle.sha256 bottle_path.sha256 => bottle_tag
+    sha256 = bottle_path.sha256
+    bottle.sha256 sha256 => bottle_tag
 
     old_spec = f.bottle_specification
     if ARGV.include?("--keep-old") && !old_spec.checksums.empty?
```

Comment:
Can we now store `--keep-old` in the json file now? so we don't need to pass it again for merging.